src/main/java/com/vizor/unreal/convert/ProtoProcessor.java

```diff
@@ -464,12 +471,21 @@ private CppEnum extractEnum(final TypesProvider provider, final EnumElement ee)
 
     private CppType ueNamedType(final String serviceName, final TypeElement el)
     {
+        // s6fix @bernst - Generate C++ safe and compilable code.
         if (el instanceof MessageElement)
-            return plain("F" + serviceName + "_" + el.name(), Struct);
+        {
+            return plain("F" + serviceName + "_" + mixedCharacterStringToCppSafe(el.name()), Struct);
```

Comment:
Should only need Pascal Case, not CppSafe (I think?) as the element names should not contain periods or dashes